File: problems/m/mult/mult.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    n = int(input())
    initial_number = None
    for _ in range(n):
        if initial_number is None:
            initial_number = int(input())
        else:
            val = int(input())
            if val % initial_number == 0:
                print(val)
                initial_number = None


if __name__ == '__main__':
    main()

File: problems/m/mult/mult.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>

using namespace std;

int main() {
    int n, value, initial_number;

    cin >> n;

    initial_number = -1;
    for (int i=0; i<n; i++) {
        cin >> value;

        if (initial_number == -1) {
            initial_number = value;
            continue;
        }
        else {
            if (value % initial_number == 0) {
                cout << value << endl;
                initial_number = -1;
            }
        }

    }
}